depthai_demo.py

```diff
@@ -244,6 +244,13 @@ def startLoop(self):
                     #mjpeg = packetData
                     #mat = cv2.imdecode(mjpeg, cv2.IMREAD_COLOR)
                     #cv2.imshow('mjpeg', mat)
+                elif packet.stream_name == 'raw_color':
```

Comment:
change "raw_color" to "rgb"

Reply:
All 3 of us discussed offline and determined probably `color` better since the output itself is YUV and then converted on host.